cider/include/cider/batch/CiderBatch.h

```diff
@@ -354,21 +354,30 @@ class CiderBatch {
 #define PRINT_BY_TYPE(C_TYPE)                \
   {                                          \
     ss << "column type: " << #C_TYPE << " "; \
-    C_TYPE* buf = (C_TYPE*)table_ptr_[i];    \
+    C_TYPE* buf = (C_TYPE*)table_ptr_[loc];  \
     for (int j = 0; j < row_num_; j++) {     \
-      ss << buf[j] << "\t";                  \
+      ss << std::to_string(buf[j]) << "\t";  \
     }                                        \
     break;                                   \
   }
+#define PRINT_BY_TYPE_PTR(C_TYPE)             \
```

Comment:
what's the difference between `PRINT_BY_TYPE` and `PRINT_BY_TYPE_PTR`?

Reply:
It's ptr of `stringstream` in `PRINT_BY_TYPE_PTR` since we cannot pass down the value of `stringstream`. It looks some kind of duplicated..